src/components/UserMenu/UserMenu.test.js

```diff
@@ -8,7 +8,7 @@ import dataDump from "testing/complete-redux-store-dump";
 import UserMenu from "./UserMenu";
 
 const mockStore = configureStore([]);
-describe("User Icon", () => {
+describe("User Menu", () => {
```

Comment:
We need one more test here... That when you click the button it calls dispatch with the appropriate values.